src/test/java/reposense/system/CommandRunnerTest.java

```diff
@@ -31,18 +32,20 @@ public void checkoutTest() {
 
     @Test
     public void logWithContentTest() {
-        String content = CommandRunner.gitLog(TestConstants.LOCAL_TEST_REPO_ADDRESS, null, null);
+        String content =
+                CommandRunner.gitLog(TestConstants.LOCAL_TEST_REPO_ADDRESS, null, null, Collections.emptyList());
```

Comment:
Should be using the `DEFAULT_FORMATS` here (and tests below), since an empty list for us here will mean `git log` will log the commits for all file types (which they tracked). 
We should also add tests for different formats given, like `*.py` will give empty content since we have no `.py` files. 
Remember to update the test name to the 3 point format.